Willning/DLOplanTool
# Dlo 42

### **Related Issue/Keyword:**

### **Description:**
Once the user has finished mapping ideas, user should be able to progress to amplifying the notes of existing ideas. 

### **Testing:**

Open a project and click the amplification arrow to be taken to the amplification screen, you should then be able to add and remove amplifications from keywords selected from the auto loaded mapping data.

Add anything in particular to look out for
make sure you click the tick box to go to the next idea

**Steps for manual testing:**

1. open an existing project
2. click the "amplification" arrow
3. click the check marks or click on the idea titles to navigate around the screen
4. ensure keywords are selected from the ideas in your project map
5. click on some key words and add amplifications to them

**note:**
the amplifications currently do not persist to the db and this will be done in the next pr (currently it simply prints to the console when it normally would persist)

### **Checklist:**
